src/backend/funmooc/templates/richie/base.html

```diff
@@ -0,0 +1,14 @@
+{% extends "richie/base.html" %}
+{% load static %}
+
+{% block meta_favicons %}
+<link rel="apple-touch-icon" sizes="180x180" href="{% static 'richie/favicon/apple-touch-icon.png' %}">
```

Comment:
We're pythonists used to underscores in file names. What's your rational for using "-"?

Reply:
Because this is a static file, not something you can cast as a variable, plus this is generated stuff.